api/management/policy.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: '3.0.0'
+
+info:
+  title: policy management
+  description: Veraison attestation policy management
+  version: '1.0.0alpha'
+
+servers:
+  - url: 'https://veraison.example/management/v1'
+
+paths:
+  /policy/{scheme}:
+    post:
+      description: >
+        Update policy rules for the scheme identified in path.
+      parameters:
+        - name: name
+          in: query
+          description: >
+            The name under which the policy will be added.
+          required: false
+          schema:
```

Comment:
```suggestion
          scheme:
```

Reply:
I believe `schema` is correct here?

Reply:
I was, however, missing the description of the scheme parameter (the checker informed me that path params must also be described), which is now added.

Reply:
sorry, yes you are right, I misinterpreted the error from the validator...